frontend/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Switch to the frontend folder
+cd frontend
+
+# Build the frontend application
+ng build --configuration=production
+
+tail -f /dev/null
```

Comment:
Is the tail needed? I thought that it just builds the frontend and then the container is done. Is there a need to keep it running endlessly?

Reply:
Fixed